mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
Which files do you want to symlink? Current list of files:

```
ROSLYN_FILES_FOR_MSBUILD = \
        $(ROSLYN_CSC_DIR)/Microsoft.Build.Tasks.CodeAnalysis.dll \
        $(ROSLYN_CSC_DIR)/Microsoft.CSharp.Core.targets \
        $(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.CSharp.Scripting.dll \
        $(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.CSharp.dll \
        $(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.Scripting.dll \
        $(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.VisualBasic.dll \
        $(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.dll \
        $(ROSLYN_CSC_DIR)/Microsoft.VisualBasic.Core.targets \
        $(ROSLYN_CSC_DIR)/System.Collections.Immutable.dll \
        $(ROSLYN_CSC_DIR)/System.Reflection.Metadata.dll \
        $(ROSLYN_CSC_DIR)/VBCSCompiler.exe \
        $(ROSLYN_CSC_DIR)/VBCSCompiler.exe.config \
        $(ROSLYN_CSC_DIR)/csi.exe \
        $(ROSLYN_CSC_DIR)/csi.exe.config \
        $(ROSLYN_CSC_DIR)/csi.rsp```

Do we want the `*Scripting`, `*VisualBasic.dll`, `*Build.Tasks*`, `csi*` and `VBCSCompiler*` assemblies in 4.5 too? Currently, we have `Microsoft.CodeAnalysis.CSharp.dll`, `Microsoft.CodeAnalysis.dll` and `System*` assemblies in 4.5, so we can symlink those.